com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -940,11 +940,25 @@ public void SetSingleton()
             OnSingletonReady?.Invoke();
         }
 
+        /// <summary>
+        /// Returns the root GameObject of the NetworkManager's GameObject
+        /// </summary>
+        internal GameObject GetRootParent(GameObject currentGameObject = null)
+        {
+            currentGameObject = currentGameObject == null ? gameObject : currentGameObject;
+            if (currentGameObject.transform.parent != null)
+            {
+                return GetRootParent(currentGameObject.transform.parent.gameObject);
```

Comment:
Can `transform.root` work here?

Reply:
Yes!  Awesome catch! I am about to check in that adjustment.